src/components/Treebank/TreebankReact.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
```

Comment:
This file is the most important change. It contains the integration with `treebank-react`.